vhdbuilder/packer/install-dependencies.sh

```diff
@@ -252,6 +252,7 @@ AMD64_ONLY_CNI_VERSIONS="
 MULTI_ARCH_VNET_CNI_VERSIONS="
 1.4.14
 1.4.21
```

Comment:
Again prety sure 1.4.21 can be removed though unless arm using it somehow. Have to check with Charlie li.

Reply:
@paulgmiller Charlie shared "1.4.21 is cached in VHD recently by networking SIG. not used in RP side so far." so I think that v1.4.21 should not be removed.